aten/src/ATen/native/Loss.cpp

```diff
@@ -280,7 +280,7 @@ Tensor& binary_cross_entropy_out_cpu(const Tensor& input, const Tensor& target,
       .add_owned_input(at::squeeze(target))
       .build();
 
-    AT_DISPATCH_FLOATING_TYPES(loss.scalar_type(), "binary_cross_entropy", [&] {
+    AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES(loss.scalar_type(), "binary_cross_entropy", [&] {
```

Comment:
why change here?

Reply:
Another reviewer told me to add support for the CPU kernels too so I made that change. I wasn't sure if this is correct.

https://github.com/pytorch/pytorch/pull/89649#pullrequestreview-1264631403

Reply:
But you are adding support to binary_cross_entropy, not silu.

Reply:
How should I add support to the CPU kernels of silu?

Reply:
Do I have to add `cpu_kernel` to `silu_kerne`l and `silu_kernel_backward`?

https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/cuda/ActivationSiluKernel.cu